app/assets/javascripts/cascade/new-home.js

```diff
@@ -5,7 +5,7 @@
         if ( onHomePage ) {
             cu_window_manager.initialize();
             cu_hero_area.initialize();
-            cu_stories_area.initialize();
+            //cu_stories_area.initialize();
```

Comment:
in javascripts/cascade/new-home.js I'm wondering why cu_stories_area is commented out now. Was there never any function that it called? In same script, there's several places where it replaces a path eg /_files with the fully qualified path such as http://www.chapman.edu path. I think now that SSL is mandatory on WWW site, those paths should be https not http. lines 422, 543, 547 etc.

Reply:
I ran into the same issue when working on the footer story. The variable `cu_stories_area` is never created so this throws an "Uncaught TypeError: Cannot read property of undefined" error

Reply:
Ok. I couldn't find it either. We definitely have an area on the page for Stories but it must not need any special Javascript. What about the changing of hardcoded http://www.chapman.edu to https instead?

Reply:
I don't recall, also can't find that change in this PRs changed files. Can you point to exactly what you mean?

Reply:
I checked the homepage on dev-www on IE, using https, and didn't get any warnings about mismatch so I guess it's ok as is.